.github/workflows/ci.yml

```diff
@@ -61,3 +64,5 @@ jobs:
           cmake -DCMAKE_CXX_FLAGS_INIT=-D_WIN32_WINNT=0x0A00 -DCMAKE_TOOLCHAIN_FILE=./conan_paths.cmake -G Ninja ..
           cmake --build . -j
         shell: bash
+      - name: Install
+        run: cd build && cmake --build . --target install
```

Comment:
Using `cmake --build build --target install` avoids the `cd` … I guess.

Reply:
... but your line is more portable, as it does not depend on the generator. Switched to your recommendation.